main.go

```diff
@@ -63,25 +62,25 @@ func main() {
 	}
 
 	// Expose Vault token metrics.
-	{
-		if tokenPath == "" {
-			panic(microerror.Maskf(invalidConfigError, "path to token folder can not be empty"))
-		}
-		if vaultURL == "" {
-			panic(microerror.Maskf(invalidConfigError, "Vault URL can not be empty"))
-		}
-
-		c := token.Config{
-			Path:     tokenPath,
-			VaultURL: vaultURL,
-		}
-
-		tokenExporter, err := token.New(c)
-		if err != nil {
-			panic(microerror.Mask(err))
-		}
-		prometheus.MustRegister(tokenExporter)
-	}
+	// {
```

Comment:
Having the vault collector in cert-exporter was 50/50 to begin with, but doesn't follow now that we're deploying cert-exporter to guest clusters.

I'm just dumping it here for now, because I'm planning on refactoring cert-exporter with exporterkit shortly, and we'll move the vault related functions to a separate exporter, tracking in https://github.com/giantswarm/giantswarm/issues/3996.